zeppelin-plugins/launcher/k8s-standard/src/main/java/org/apache/zeppelin/interpreter/launcher/K8sStandardInterpreterLauncher.java

```diff
@@ -73,6 +73,11 @@ String getNamespace() throws IOException {
     if (isRunningOnKubernetes()) {
       return readFile(Config.KUBERNETES_NAMESPACE_PATH, Charset.defaultCharset()).trim();
     } else {
+      //namespace may be set from the interpreter properties
```

Comment:
This feature should also be available with a Zeppelin server running on Kubernetes.